javascript/javascript_sdk/src/sensr_client.js

```diff
@@ -1,11 +1,21 @@
 const WebSocket = require('ws');
 const outputMsg = require('./../js_proto/sensr_proto/output_pb.js');
 const pointMsg = require('./../js_proto/sensr_proto/point_cloud_pb');
+const fs = require('fs');
+
+const cert_location = __dirname + '/../../../' + 'keys/sensr-sdk-ca.crt'
```

Comment:
'/../../../' :feelsbadman: but I can't think of any cleaner way of doing this.

Reply:
Yes, sorry about this ugly style.